x-pack/plugins/apm/public/components/shared/Summary/CompositeSpanDurationSummaryItem.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiToolTip, EuiText } from '@elastic/eui';
+import { asDuration } from '../../../../common/utils/formatters';
+
+interface Props {
+  count: number;
+  durationSum: number;
+}
+
+function CompositeSpanDurationSummaryItem({ count, durationSum }: Props) {
+  const avgDuration = durationSum / count;
+
+  const label = i18n.translate('xpack.apm.compositeSpanDurationLabel', {
+    defaultMessage: 'Average duration',
+  });
+  const callsLabel = i18n.translate('xpack.apm.compositeSpanCallsLabel', {
+    defaultMessage: 'calls',
+  });
+  return (
+    <>
+      <EuiToolTip content={label}>
+        <EuiText>{count} {callsLabel}, &#8709; {asDuration(avgDuration)}</EuiText>
```

Comment:
What is this `&#8709;`?

Maybe you could have done something like:
```suggestion
        <EuiText>{
                i18n.translate('xpack.apm.compositeSpanCallsLabel', {
                defaultMessage: '{cont} calls {duration}',
                values: {
                  count,
                  duration: asDuration(avgDuration)
          })
        }</EuiText>
```

Reply:
∅?

Reply:
> ∅?
My wrong assumption that this symbol is also internationally known as the symbol for average. 
TIL: This is only in German-speaking countries used as the average symbol. :-)